.script/test

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
This will tell bash to exit immediately if a command exits with a non-zero status.

I added a comment.